core/tee/tee_rpmb_fs.c

```diff
@@ -1086,6 +1091,9 @@ static TEE_Result tee_rpmb_init(uint16_t dev_id)
 	struct rpmb_dev_info dev_info;
 	uint32_t nblocks = 0;
 
+	if (rpmb_dead)
```

Comment:
Can this happen?

Reply:
Yes, this function is called at the start of all the RPMB entry functions.

Reply:
Ah, yes.